In [1]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns
%matplotlib inline

import warnings
warnings.filterwarnings(action='ignore')

# 한글폰트
from matplotlib import font_manager, rc
f_path = "c:/Windows/Fonts/malgun.ttf"
font_name = font_manager.FontProperties(fname=f_path).get_name()
rc('font', family=font_name)

from IPython.display import set_matplotlib_formats
set_matplotlib_formats('retina')


pd.options.display.max_rows=100
pd.options.display.max_columns=100
# pd.set_option(‘display.float_format’,‘{:.4f}’.format)

# 품명 확인

In [40]:
df = pd.read_csv('./data/rawdata_Final.csv')
set(df['product_name'])

{'나이키 덩크로우 골든로드',
 '나이키 덩크로우 라이트본',
 '나이키 덩크로우 바시티그린',
 '나이키 덩크로우 범고래',
 '나이키 덩크로우 유니버시티블루',
 '나이키 덩크로우 코스트',
 '나이키 덩크하이 네이비',
 '나이키 덩크하이 범고래',
 '나이키 덩크하이 오렌지',
 '뉴발란스 992 그레이',
 '뉴발란스 992 네이비',
 '뉴발란스 992 블랙그레이',
 '뉴발란스 992 화이트실버',
 '뉴발란스 993 그레이',
 '뉴발란스 993 네이비',
 '뉴발란스 993 블랙',
 '조던 1 로우 스타피쉬',
 '조던 1 로우 울프그레이',
 '조던 1 로우 트레비스 스캇',
 '조던1 미드 그레이포그',
 '조던1 미드 스모크그레이',
 '조던1 미드 울프그레이',
 '조던1 미드 짐레드',
 '조던1 하이 다크모카',
 '조던1 하이 스모크그레이',
 '조던1 하이 하이퍼로얄'}

# 네이버 데이터랩 - 검색어트렌드
https://datalab.naver.com/keyword/trendResult.naver?hashKey=N_d50e90e8ba8d94213a3d4495652d3606

## 나이키 덩크 
### 로우
골든로드    
라이트본     
바시티그린     
범고래      
유니버시티블루      
코스트

### 하이
네이비    
범고래     
오렌지

### 덩크 로우 데이터셋

In [58]:
# 덩크로우 데이터셋
D_low_1 = pd.read_excel('./data/NAVER-trend/trend_dunk_low1.xlsx')
D_low_2 = pd.read_excel('./data/NAVER-trend/trend_dunk_low2.xlsx')

In [69]:
D_low_1 = D_low_1.drop(['날짜.1','날짜.2', '날짜.3', '날짜.4' ], axis=1)
D_low_1

KeyError: "['날짜.1' '날짜.2' '날짜.3' '날짜.4'] not found in axis"

In [62]:
D_low_1.index = D_low_1['날짜']
D_low_1 = D_low_1.drop(['날짜'], axis=1)
D_low_1

나이키 덩크 로우 골든로드   나이키 덩크 로우 라이트본  나이키 덩크 로우 바시티그린  나이키 덩크 로우 범고래  \
날짜                                                                            
2020-11-01          0.00000         0.00000          0.00000        0.00000   
2020-11-02          0.00000         0.00000          0.00000        0.00000   
2020-11-03          0.00000         0.00000          0.00000        0.00000   
2020-11-04          0.00000         0.00000          0.00000        0.00000   
2020-11-05          0.00000         0.00000          0.00000        0.02591   
...                     ...             ...              ...            ...   
2021-11-09          0.18141         4.95011          0.12526       16.83296   
2021-11-10          0.18141         4.80324          0.07775        9.29117   
2021-11-11          0.14686         4.68230          0.11662        8.42728   
2021-11-12          0.22029         3.79681          0.06479        8.86786   
2021-11-13          0.13390         3.54196          0.09934        8.71668   

            나이키 덩크 로우 유니버시티블루  
날짜                             
2020-11-01            0.00000  
2020-11-02            0.00000  
2020-11-03            0.00000  
2020-11-04            0.00000  
2020-11-05            0.00000  
...                       ...  
2021-11-09            0.09070  
2021-11-10            0.11230  
2021-11-11            0.09934  
2021-11-12            0.13390  
2021-11-13            0.10366  

[378 rows x 5 columns]

In [65]:
D_low_2.index = D_low_2['날짜']
D_low_2 = D_low_2.drop(['날짜'], axis=1)
D_low_2

나이키 덩크로우 코스트
날짜                      
2020-11-01       0.00000
2020-11-02       0.00000
2020-11-03       0.00000
2020-11-04       0.00000
2020-11-05       0.00000
...                  ...
2021-11-09       5.83501
2021-11-10       6.23742
2021-11-11       4.42655
2021-11-12       4.42655
2021-11-13       4.22535

[378 rows x 1 columns]

In [67]:
D_low = pd.merge(D_low_1, D_low_2, left_index=True, right_index=True, how='outer')
D_low
D_low.to_csv('./data/NAVER-trend/trend_dunk_low.csv')

In [82]:
D_low.max()

나이키 덩크 로우 골든로드         1.74506
나이키 덩크 로우 라이트본         6.00837
나이키 덩크 로우 바시티그린        0.99347
나이키 덩크 로우 범고래        100.00000
나이키 덩크 로우 유니버시티블루      2.26340
나이키 덩크로우 코스트         100.00000
dtype: float64

### 덩크 하이 데이터셋

In [84]:
D_high = pd.read_excel('./data/NAVER-trend/trend_dunk_high.xlsx')
D_high

날짜  나이키 덩크하이 네이비        날짜.1  나이키 덩크하이 범고래        날짜.2  \
0    2020-11-01       0.00000  2020-11-01       0.00000  2020-11-01   
1    2020-11-02       0.00000  2020-11-02       0.00000  2020-11-02   
2    2020-11-03       0.00000  2020-11-03       0.00000  2020-11-03   
3    2020-11-04       0.00000  2020-11-04       0.00000  2020-11-04   
4    2020-11-05       0.00000  2020-11-05       0.00000  2020-11-05   
..          ...           ...         ...           ...         ...   
373  2021-11-09       0.45248  2021-11-09       5.07792  2021-11-09   
374  2021-11-10       0.85470  2021-11-10       4.12267  2021-11-10   
375  2021-11-11       0.65359  2021-11-11       4.47461  2021-11-11   
376  2021-11-12       0.60331  2021-11-12       5.12820  2021-11-12   
377  2021-11-13       0.55304  2021-11-13       3.92156  2021-11-13   

     나이키 덩크하이 오렌지  
0             0.0  
1             0.0  
2             0.0  
3             0.0  
4             0.0  
..            ...  
373           0.0  
374           0.0  
375           0.0  
376           0.0  
377           0.0  

[378 rows x 6 columns]

In [85]:
D_high = D_high.drop(['날짜.1', '날짜.2'], axis=1)
D_high

날짜  나이키 덩크하이 네이비  나이키 덩크하이 범고래  나이키 덩크하이 오렌지
0    2020-11-01       0.00000       0.00000           0.0
1    2020-11-02       0.00000       0.00000           0.0
2    2020-11-03       0.00000       0.00000           0.0
3    2020-11-04       0.00000       0.00000           0.0
4    2020-11-05       0.00000       0.00000           0.0
..          ...           ...           ...           ...
373  2021-11-09       0.45248       5.07792           0.0
374  2021-11-10       0.85470       4.12267           0.0
375  2021-11-11       0.65359       4.47461           0.0
376  2021-11-12       0.60331       5.12820           0.0
377  2021-11-13       0.55304       3.92156           0.0

[378 rows x 4 columns]

In [86]:
D_high.index = D_high['날짜']
D_high

날짜  나이키 덩크하이 네이비  나이키 덩크하이 범고래  나이키 덩크하이 오렌지
날짜                                                              
2020-11-01  2020-11-01       0.00000       0.00000           0.0
2020-11-02  2020-11-02       0.00000       0.00000           0.0
2020-11-03  2020-11-03       0.00000       0.00000           0.0
2020-11-04  2020-11-04       0.00000       0.00000           0.0
2020-11-05  2020-11-05       0.00000       0.00000           0.0
...                ...           ...           ...           ...
2021-11-09  2021-11-09       0.45248       5.07792           0.0
2021-11-10  2021-11-10       0.85470       4.12267           0.0
2021-11-11  2021-11-11       0.65359       4.47461           0.0
2021-11-12  2021-11-12       0.60331       5.12820           0.0
2021-11-13  2021-11-13       0.55304       3.92156           0.0

[378 rows x 4 columns]

In [88]:
D_high = D_high.drop(['날짜'], axis=1)
D_high

나이키 덩크하이 네이비  나이키 덩크하이 범고래  나이키 덩크하이 오렌지
날짜                                                  
2020-11-01       0.00000       0.00000           0.0
2020-11-02       0.00000       0.00000           0.0
2020-11-03       0.00000       0.00000           0.0
2020-11-04       0.00000       0.00000           0.0
2020-11-05       0.00000       0.00000           0.0
...                  ...           ...           ...
2021-11-09       0.45248       5.07792           0.0
2021-11-10       0.85470       4.12267           0.0
2021-11-11       0.65359       4.47461           0.0
2021-11-12       0.60331       5.12820           0.0
2021-11-13       0.55304       3.92156           0.0

[378 rows x 3 columns]

In [89]:
D_high.to_csv('./data/NAVER-trend/trend_dunk_high.csv')

## 뉴발란스
### 992
그레이   
네이비   
블랙그레이   
화이트실버 

### 993
그레이    
네이비    
블랙    

### 뉴발란스 992 데이터셋

In [90]:
new992 = pd.read_excel('./data/NAVER-trend/trend_new992.xlsx')
new992

날짜  뉴발란스 992 그레이        날짜.1  뉴발란스 992 네이비        날짜.2  \
0    2020-11-01       7.47217  2020-11-01       4.53100  2020-11-01   
1    2020-11-02       9.37996  2020-11-02       3.57710  2020-11-02   
2    2020-11-03       9.77742  2020-11-03       3.17965  2020-11-03   
3    2020-11-04      12.48012  2020-11-04       4.21303  2020-11-04   
4    2020-11-05      11.44674  2020-11-05       2.22575  2020-11-05   
..          ...           ...         ...           ...         ...   
373  2021-11-09      18.44197  2021-11-09       5.72337  2021-11-09   
374  2021-11-10      20.90620  2021-11-10       5.48489  2021-11-10   
375  2021-11-11      18.44197  2021-11-11       4.29252  2021-11-11   
376  2021-11-12      18.04451  2021-11-12       5.00794  2021-11-12   
377  2021-11-13      21.62162  2021-11-13       4.68998  2021-11-13   

     뉴발란스 992 블랙그레이        날짜.3  뉴발란스 992 화이트실버  
0           0.00000  2020-11-01         0.71542  
1           0.00000  2020-11-02         0.63593  
2           0.00000  2020-11-03         0.47694  
3           0.00000  2020-11-04         0.39745  
4           0.00000  2020-11-05         0.55643  
..              ...         ...             ...  
373         2.62321  2021-11-09         2.22575  
374         2.70270  2021-11-10         2.78219  
375         2.78219  2021-11-11         1.11287  
376         2.78219  2021-11-12         2.46422  
377         2.38473  2021-11-13         1.27186  

[378 rows x 8 columns]

In [93]:
new992 = new992.drop(['날짜.1', '날짜.2', '날짜.3'], axis=1)
new992.index = new992['날짜']


날짜  뉴발란스 992 그레이  뉴발란스 992 네이비  뉴발란스 992 블랙그레이  \
날짜                                                                   
2020-11-01  2020-11-01       7.47217       4.53100         0.00000   
2020-11-02  2020-11-02       9.37996       3.57710         0.00000   
2020-11-03  2020-11-03       9.77742       3.17965         0.00000   
2020-11-04  2020-11-04      12.48012       4.21303         0.00000   
2020-11-05  2020-11-05      11.44674       2.22575         0.00000   
...                ...           ...           ...             ...   
2021-11-09  2021-11-09      18.44197       5.72337         2.62321   
2021-11-10  2021-11-10      20.90620       5.48489         2.70270   
2021-11-11  2021-11-11      18.44197       4.29252         2.78219   
2021-11-12  2021-11-12      18.04451       5.00794         2.78219   
2021-11-13  2021-11-13      21.62162       4.68998         2.38473   

            뉴발란스 992 화이트실버  
날짜                          
2020-11-01         0.71542  
2020-11-02         0.63593  
2020-11-03         0.47694  
2020-11-04         0.39745  
2020-11-05         0.55643  
...                    ...  
2021-11-09         2.22575  
2021-11-10         2.78219  
2021-11-11         1.11287  
2021-11-12         2.46422  
2021-11-13         1.27186  

[378 rows x 5 columns]

In [95]:
new992 = new992.drop(['날짜'], axis=1)
new992

뉴발란스 992 그레이  뉴발란스 992 네이비  뉴발란스 992 블랙그레이  뉴발란스 992 화이트실버
날짜                                                                    
2020-11-01       7.47217       4.53100         0.00000         0.71542
2020-11-02       9.37996       3.57710         0.00000         0.63593
2020-11-03       9.77742       3.17965         0.00000         0.47694
2020-11-04      12.48012       4.21303         0.00000         0.39745
2020-11-05      11.44674       2.22575         0.00000         0.55643
...                  ...           ...             ...             ...
2021-11-09      18.44197       5.72337         2.62321         2.22575
2021-11-10      20.90620       5.48489         2.70270         2.78219
2021-11-11      18.44197       4.29252         2.78219         1.11287
2021-11-12      18.04451       5.00794         2.78219         2.46422
2021-11-13      21.62162       4.68998         2.38473         1.27186

[378 rows x 4 columns]

In [96]:
new992.to_csv('./data/NAVER-trend/trend_new992.csv')

### 뉴발란스 993 데이터셋

In [97]:
new993 = pd.read_excel('./data/NAVER-trend/trend_new993.xlsx')
new993

날짜  뉴발란스 993 그레이        날짜.1  뉴발란스 993 네이비        날짜.2  \
0    2020-11-01      31.99268  2020-11-01       7.49542  2020-11-01   
1    2020-11-02      23.76599  2020-11-02       7.12979  2020-11-02   
2    2020-11-03      30.53016  2020-11-03       8.04387  2020-11-03   
3    2020-11-04      34.18647  2020-11-04       7.67824  2020-11-04   
4    2020-11-05      36.92870  2020-11-05       6.76416  2020-11-05   
..          ...           ...         ...           ...         ...   
373  2021-11-09      67.64168  2021-11-09       5.66727  2021-11-09   
374  2021-11-10      57.22120  2021-11-10       2.74223  2021-11-10   
375  2021-11-11      65.44789  2021-11-11       6.39853  2021-11-11   
376  2021-11-12      62.88848  2021-11-12       4.57038  2021-11-12   
377  2021-11-13      66.54478  2021-11-13       6.21572  2021-11-13   

     뉴발란스 993 블랙  
0       14.99085  
1       13.34552  
2       24.49725  
3       20.47531  
4       18.46435  
..           ...  
373     20.29250  
374     12.61425  
375     12.61425  
376     10.78610  
377     10.60329  

[378 rows x 6 columns]

In [98]:
new993.index = new993['날짜']
new993 = new993.drop(['날짜','날짜.1', '날짜.2'], axis=1)
new993

뉴발란스 993 그레이  뉴발란스 993 네이비  뉴발란스 993 블랙
날짜                                                 
2020-11-01      31.99268       7.49542     14.99085
2020-11-02      23.76599       7.12979     13.34552
2020-11-03      30.53016       8.04387     24.49725
2020-11-04      34.18647       7.67824     20.47531
2020-11-05      36.92870       6.76416     18.46435
...                  ...           ...          ...
2021-11-09      67.64168       5.66727     20.29250
2021-11-10      57.22120       2.74223     12.61425
2021-11-11      65.44789       6.39853     12.61425
2021-11-12      62.88848       4.57038     10.78610
2021-11-13      66.54478       6.21572     10.60329

[378 rows x 3 columns]

In [99]:
new993.to_csv('./data/NAVER-trend/trend_new993.csv')

## 조던1
### 로우
스타피쉬    
울프그레이    
트레비스 스캇    

### 미드
그레이포크    
스모크그레이     
울프그레이     
짐레드   

### 하이
다크모카     
스모크그레이     
하이퍼로얄

### 조던1 로우 데이터셋

In [100]:
jordan_law = pd.read_excel('./data/NAVER-trend/trend_jordan1_low.xlsx')
jordan_law

날짜  조던1 로우 스타피쉬        날짜.1  조던1 로우 울프그레이        날짜.2  \
0    2020-11-01      0.00000  2020-11-01       0.00000  2020-11-01   
1    2020-11-02      0.00000  2020-11-02       0.00000  2020-11-02   
2    2020-11-03      0.00000  2020-11-03       0.00000  2020-11-03   
3    2020-11-04      0.00000  2020-11-04       0.00000  2020-11-04   
4    2020-11-05      0.00000  2020-11-05       0.00000  2020-11-05   
..          ...          ...         ...           ...         ...   
373  2021-11-09      3.60205  2021-11-09       1.20068  2021-11-09   
374  2021-11-10      2.74442  2021-11-10       1.02915  2021-11-10   
375  2021-11-11      3.08747  2021-11-11       1.02915  2021-11-11   
376  2021-11-12      4.11663  2021-11-12       2.74442  2021-11-12   
377  2021-11-13      4.63121  2021-11-13       1.71526  2021-11-13   

     조던1 로우 트레비스 스캇  
0               0.0  
1               0.0  
2               0.0  
3               0.0  
4               0.0  
..              ...  
373             0.0  
374             0.0  
375             0.0  
376             0.0  
377             0.0  

[378 rows x 6 columns]

In [102]:
jordan_law.index = jordan_law['날짜']
jordan_law = jordan_law.drop(['날짜', '날짜.1', '날짜.2'], axis=1)
jordan_law

조던1 로우 스타피쉬  조던1 로우 울프그레이  조던1 로우 트레비스 스캇
날짜                                                   
2020-11-01      0.00000       0.00000             0.0
2020-11-02      0.00000       0.00000             0.0
2020-11-03      0.00000       0.00000             0.0
2020-11-04      0.00000       0.00000             0.0
2020-11-05      0.00000       0.00000             0.0
...                 ...           ...             ...
2021-11-09      3.60205       1.20068             0.0
2021-11-10      2.74442       1.02915             0.0
2021-11-11      3.08747       1.02915             0.0
2021-11-12      4.11663       2.74442             0.0
2021-11-13      4.63121       1.71526             0.0

[378 rows x 3 columns]

In [103]:
jordan_law.to_csv('./data/NAVER-trend/trend_jordan1_low.csv')

### 조던1 미드 데이터셋

In [104]:
jordan_mid = pd.read_excel('./data/NAVER-trend/trend_jordan1_mid.xlsx')
jordan_mid

날짜  조던1 미드 그레이포크        날짜.1  조던1 미드 스모크그레이        날짜.2  \
0    2020-11-01             0  2020-11-01        3.46938  2020-11-01   
1    2020-11-02             0  2020-11-02        3.57142  2020-11-02   
2    2020-11-03             0  2020-11-03        2.85714  2020-11-03   
3    2020-11-04             0  2020-11-04        3.67346  2020-11-04   
4    2020-11-05             0  2020-11-05        2.95918  2020-11-05   
..          ...           ...         ...            ...         ...   
373  2021-11-09             0  2021-11-09        2.24489  2021-11-09   
374  2021-11-10             0  2021-11-10        1.63265  2021-11-10   
375  2021-11-11             0  2021-11-11        2.14285  2021-11-11   
376  2021-11-12             0  2021-11-12        2.24489  2021-11-12   
377  2021-11-13             0  2021-11-13        2.44897  2021-11-13   

     조던1 미드 울프그레이        날짜.3  조던1 미드 짐레드  
0         0.00000  2020-11-01     0.00000  
1         0.00000  2020-11-02     0.61224  
2         0.00000  2020-11-03     0.91836  
3         0.00000  2020-11-04     0.51020  
4         0.00000  2020-11-05     0.51020  
..            ...         ...         ...  
373       0.81632  2021-11-09     4.28571  
374       3.16326  2021-11-10     4.18367  
375       1.12244  2021-11-11     4.69387  
376       1.02040  2021-11-12     3.67346  
377       1.93877  2021-11-13     6.32653  

[378 rows x 8 columns]

In [107]:
# jordan_mid.index = jordan_mid['날짜']
# jordan_mid = jordan_mid.drop(['날짜', '날짜.1', '날짜.2'], axis=1)
jordan_mid = jordan_mid.drop(['날짜.3'], axis=1)
jordan_mid

조던1 미드 그레이포크  조던1 미드 스모크그레이  조던1 미드 울프그레이  조던1 미드 짐레드
날짜                                                               
2020-11-01             0        3.46938       0.00000     0.00000
2020-11-02             0        3.57142       0.00000     0.61224
2020-11-03             0        2.85714       0.00000     0.91836
2020-11-04             0        3.67346       0.00000     0.51020
2020-11-05             0        2.95918       0.00000     0.51020
...                  ...            ...           ...         ...
2021-11-09             0        2.24489       0.81632     4.28571
2021-11-10             0        1.63265       3.16326     4.18367
2021-11-11             0        2.14285       1.12244     4.69387
2021-11-12             0        2.24489       1.02040     3.67346
2021-11-13             0        2.44897       1.93877     6.32653

[378 rows x 4 columns]

In [108]:
jordan_mid.to_csv('./data/NAVER-trend/trend_jordan1_mid.csv')

### 조던1 하이 데이터셋

In [109]:
jordan_high = pd.read_excel('./data/NAVER-trend/trend_jordan1_high.xlsx')
jordan_high

날짜  조던1 하이 다크모카        날짜.1  조던1 하이 스모크그레이        날짜.2  \
0    2020-11-01      0.00000  2020-11-01        7.14285  2020-11-01   
1    2020-11-02      0.00000  2020-11-02        9.52380  2020-11-02   
2    2020-11-03      0.00000  2020-11-03        5.95238  2020-11-03   
3    2020-11-04      8.33333  2020-11-04        7.14285  2020-11-04   
4    2020-11-05      0.00000  2020-11-05        0.00000  2020-11-05   
..          ...          ...         ...            ...         ...   
373  2021-11-09      0.00000  2021-11-09        0.00000  2021-11-09   
374  2021-11-10      0.00000  2021-11-10        0.00000  2021-11-10   
375  2021-11-11      0.00000  2021-11-11        0.00000  2021-11-11   
376  2021-11-12      0.00000  2021-11-12        0.00000  2021-11-12   
377  2021-11-13      0.00000  2021-11-13        0.00000  2021-11-13   

     조던1 하이 하이퍼로얄  
0             0.0  
1             0.0  
2             0.0  
3             0.0  
4             0.0  
..            ...  
373           0.0  
374           0.0  
375           0.0  
376           0.0  
377           0.0  

[378 rows x 6 columns]

In [110]:
jordan_high.index = jordan_high['날짜']
jordan_high = jordan_high.drop(['날짜', '날짜.1', '날짜.2'], axis=1)
jordan_high

조던1 하이 다크모카  조던1 하이 스모크그레이  조던1 하이 하이퍼로얄
날짜                                                  
2020-11-01      0.00000        7.14285           0.0
2020-11-02      0.00000        9.52380           0.0
2020-11-03      0.00000        5.95238           0.0
2020-11-04      8.33333        7.14285           0.0
2020-11-05      0.00000        0.00000           0.0
...                 ...            ...           ...
2021-11-09      0.00000        0.00000           0.0
2021-11-10      0.00000        0.00000           0.0
2021-11-11      0.00000        0.00000           0.0
2021-11-12      0.00000        0.00000           0.0
2021-11-13      0.00000        0.00000           0.0

[378 rows x 3 columns]

In [111]:
jordan_high.to_csv('./data/NAVER-trend/trend_jordan1_high.csv')

## 네이버 데이터랩 트렌드 통합 데이터셋
jordan_high, jordan_mid, jordan_law, D_high, D_low, new992, new993

In [118]:
NAVER_trend = pd.merge(jordan_high, jordan_mid, left_index=True, right_index=True, how='outer')
NAVER_trend = pd.merge(NAVER_trend, jordan_law, left_index=True, right_index=True, how='outer')
NAVER_trend = pd.merge(NAVER_trend, D_high, left_index=True, right_index=True, how='outer')
NAVER_trend = pd.merge(NAVER_trend, D_low, left_index=True, right_index=True, how='outer')
NAVER_trend = pd.merge(NAVER_trend, new992, left_index=True, right_index=True, how='outer')
NAVER_trend = pd.merge(NAVER_trend, new993, left_index=True, right_index=True, how='outer')
NAVER_trend

조던1 하이 다크모카  조던1 하이 스모크그레이  조던1 하이 하이퍼로얄  조던1 미드 그레이포크  \
날짜                                                                   
2020-11-01      0.00000        7.14285           0.0             0   
2020-11-02      0.00000        9.52380           0.0             0   
2020-11-03      0.00000        5.95238           0.0             0   
2020-11-04      8.33333        7.14285           0.0             0   
2020-11-05      0.00000        0.00000           0.0             0   
...                 ...            ...           ...           ...   
2021-11-09      0.00000        0.00000           0.0             0   
2021-11-10      0.00000        0.00000           0.0             0   
2021-11-11      0.00000        0.00000           0.0             0   
2021-11-12      0.00000        0.00000           0.0             0   
2021-11-13      0.00000        0.00000           0.0             0   

            조던1 미드 스모크그레이  조던1 미드 울프그레이  조던1 미드 짐레드  조던1 로우 스타피쉬  \
날짜                                                                 
2020-11-01        3.46938       0.00000     0.00000      0.00000   
2020-11-02        3.57142       0.00000     0.61224      0.00000   
2020-11-03        2.85714       0.00000     0.91836      0.00000   
2020-11-04        3.67346       0.00000     0.51020      0.00000   
2020-11-05        2.95918       0.00000     0.51020      0.00000   
...                   ...           ...         ...          ...   
2021-11-09        2.24489       0.81632     4.28571      3.60205   
2021-11-10        1.63265       3.16326     4.18367      2.74442   
2021-11-11        2.14285       1.12244     4.69387      3.08747   
2021-11-12        2.24489       1.02040     3.67346      4.11663   
2021-11-13        2.44897       1.93877     6.32653      4.63121   

            조던1 로우 울프그레이  조던1 로우 트레비스 스캇  나이키 덩크하이 네이비  나이키 덩크하이 범고래  \
날짜                                                                     
2020-11-01       0.00000             0.0       0.00000       0.00000   
2020-11-02       0.00000             0.0       0.00000       0.00000   
2020-11-03       0.00000             0.0       0.00000       0.00000   
2020-11-04       0.00000             0.0       0.00000       0.00000   
2020-11-05       0.00000             0.0       0.00000       0.00000   
...                  ...             ...           ...           ...   
2021-11-09       1.20068             0.0       0.45248       5.07792   
2021-11-10       1.02915             0.0       0.85470       4.12267   
2021-11-11       1.02915             0.0       0.65359       4.47461   
2021-11-12       2.74442             0.0       0.60331       5.12820   
2021-11-13       1.71526             0.0       0.55304       3.92156   

            나이키 덩크하이 오렌지  나이키 덩크 로우 골든로드   나이키 덩크 로우 라이트본  나이키 덩크 로우 바시티그린  \
날짜                                                                           
2020-11-01           0.0          0.00000         0.00000          0.00000   
2020-11-02           0.0          0.00000         0.00000          0.00000   
2020-11-03           0.0          0.00000         0.00000          0.00000   
2020-11-04           0.0          0.00000         0.00000          0.00000   
2020-11-05           0.0          0.00000         0.00000          0.00000   
...                  ...              ...             ...              ...   
2021-11-09           0.0          0.18141         4.95011          0.12526   
2021-11-10           0.0          0.18141         4.80324          0.07775   
2021-11-11           0.0          0.14686         4.68230          0.11662   
2021-11-12           0.0          0.22029         3.79681          0.06479   
2021-11-13           0.0          0.13390         3.54196          0.09934   

            나이키 덩크 로우 범고래  나이키 덩크 로우 유니버시티블루  나이키 덩크로우 코스트  뉴발란스 992 그레이  \
날짜                                                                         
2020-11-01        0.00000            0.00000       0.00000       7.47217   
2020-11-02        0.00000           

In [119]:
NAVER_trend.to_csv('./data/NAVER-trend/NAVER_trend.csv')

# 네이버 데이터랩 - 쇼핑인사이트 - 검색어 통계
https://datalab.naver.com/shoppingInsight/sKeyword.naver

## 나이키 덩크 로우
골든로드    
라이트본     
바시티그린     
범고래      
유니버시티블루      
코스트

In [13]:
shopping_dunklow_1 = pd.read_csv('./data/naver_shoppingInsight_dunklow_1.csv')
shopping_dunklow_2 = pd.read_csv('./data/naver_shoppingInsight_dunklow_2.csv')
shopping_dunklow_1

날짜  나이키덩크로우골든로드  나이키덩크로우라이트본  나이키덩크로우바시티그린  나이키덩크로우범고래  \
0    2020-11-13          NaN          NaN           NaN         NaN   
1    2020-11-18          NaN          NaN           NaN         NaN   
2    2020-11-22          NaN          NaN           NaN         NaN   
3    2020-11-25          NaN          NaN           NaN         NaN   
4    2020-11-26          NaN          NaN           NaN         NaN   
..          ...          ...          ...           ...         ...   
342  2021-11-09          NaN          4.0           NaN         8.0   
343  2021-11-10          NaN          3.0           NaN         6.0   
344  2021-11-11          NaN          3.0           NaN         7.0   
345  2021-11-12          NaN          2.0           NaN         6.0   
346  2021-11-13          NaN          2.0           NaN         6.0   

     나이키덩크로우유니버시티블루  
0               NaN  
1               NaN  
2               NaN  
3               NaN  
4               NaN  
..              ...  
342             NaN  
343             NaN  
344             NaN  
345             NaN  
346             NaN  

[347 rows x 6 columns]

In [18]:
shopping_dunklow_1.index = shopping_dunklow_1['날짜']
shopping_dunklow_1 = shopping_dunklow_1.drop(['날짜'], axis=1)
shopping_dunklow_1

나이키덩크로우골든로드  나이키덩크로우라이트본  나이키덩크로우바시티그린  나이키덩크로우범고래  나이키덩크로우유니버시티블루
날짜                                                                            
2020-11-13          NaN          NaN           NaN         NaN             NaN
2020-11-18          NaN          NaN           NaN         NaN             NaN
2020-11-22          NaN          NaN           NaN         NaN             NaN
2020-11-25          NaN          NaN           NaN         NaN             NaN
2020-11-26          NaN          NaN           NaN         NaN             NaN
...                 ...          ...           ...         ...             ...
2021-11-09          NaN          4.0           NaN         8.0             NaN
2021-11-10          NaN          3.0           NaN         6.0             NaN
2021-11-11          NaN          3.0           NaN         7.0             NaN
2021-11-12          NaN          2.0           NaN         6.0             NaN
2021-11-13          NaN          2.0           NaN         6.0             NaN

[347 rows x 5 columns]

In [19]:
shopping_dunklow_2.index = shopping_dunklow_2['날짜']
shopping_dunklow_2 = shopping_dunklow_2.drop(['날짜'], axis=1)
shopping_dunklow_2

나이키덩크로우코스트
날짜                    
2020-12-28           2
2021-01-01           1
2021-01-02           9
2021-01-03           3
2021-01-04          10
...                ...
2021-11-09          10
2021-11-10           9
2021-11-11          20
2021-11-12          11
2021-11-13           3

[318 rows x 1 columns]

In [41]:
shopping_dunklow = pd.merge(shopping_dunklow_1, shopping_dunklow_2, left_index=True, right_index=True, how='outer')
shopping_dunklow.to_csv('./data/shopping_dunklow.csv')
shopping_dunklow

나이키덩크로우골든로드  나이키덩크로우라이트본  나이키덩크로우바시티그린  나이키덩크로우범고래  \
날짜                                                               
2020-11-13          NaN          NaN           NaN         NaN   
2020-11-18          NaN          NaN           NaN         NaN   
2020-11-22          NaN          NaN           NaN         NaN   
2020-11-25          NaN          NaN           NaN         NaN   
2020-11-26          NaN          NaN           NaN         NaN   
...                 ...          ...           ...         ...   
2021-11-09          NaN          4.0           NaN         8.0   
2021-11-10          NaN          3.0           NaN         6.0   
2021-11-11          NaN          3.0           NaN         7.0   
2021-11-12          NaN          2.0           NaN         6.0   
2021-11-13          NaN          2.0           NaN         6.0   

            나이키덩크로우유니버시티블루  나이키덩크로우코스트  
날짜                                      
2020-11-13             NaN         NaN  
2020-11-18             NaN         NaN  
2020-11-22             NaN         NaN  
2020-11-25             NaN         NaN  
2020-11-26             NaN         NaN  
...                    ...         ...  
2021-11-09             NaN        10.0  
2021-11-10             NaN         9.0  
2021-11-11             NaN        20.0  
2021-11-12             NaN        11.0  
2021-11-13             NaN         3.0  

[347 rows x 6 columns]

## 나이키 덩크 하이
네이비    
범고래     
오렌지

In [42]:
sample1 = pd.read_csv('./data/shopping_dunkhigh.csv')
sample1

날짜  나이키덩크하이네이비  나이키덩크하이범고래  나이키덩크하이오렌지
0    2020-11-15         NaN         NaN         NaN
1    2020-11-17         NaN         NaN         NaN
2    2020-11-20         1.0         NaN         NaN
3    2020-11-25         NaN         NaN         NaN
4    2020-12-03         NaN         NaN         NaN
..          ...         ...         ...         ...
326  2021-11-09         2.0        10.0         NaN
327  2021-11-10         3.0         9.0         1.0
328  2021-11-11         1.0         9.0         NaN
329  2021-11-12         1.0        10.0         NaN
330  2021-11-13         1.0         6.0         NaN

[331 rows x 4 columns]

In [43]:
sample1.index = sample1['날짜']
sample1 = sample1.drop(['날짜'], axis=1)
sample1

나이키덩크하이네이비  나이키덩크하이범고래  나이키덩크하이오렌지
날짜                                            
2020-11-15         NaN         NaN         NaN
2020-11-17         NaN         NaN         NaN
2020-11-20         1.0         NaN         NaN
2020-11-25         NaN         NaN         NaN
2020-12-03         NaN         NaN         NaN
...                ...         ...         ...
2021-11-09         2.0        10.0         NaN
2021-11-10         3.0         9.0         1.0
2021-11-11         1.0         9.0         NaN
2021-11-12         1.0        10.0         NaN
2021-11-13         1.0         6.0         NaN

[331 rows x 3 columns]

In [45]:
sample1.to_csv('./data/shopping_dunkhigh.csv')
sample1

나이키덩크하이네이비  나이키덩크하이범고래  나이키덩크하이오렌지
날짜                                            
2020-11-15         NaN         NaN         NaN
2020-11-17         NaN         NaN         NaN
2020-11-20         1.0         NaN         NaN
2020-11-25         NaN         NaN         NaN
2020-12-03         NaN         NaN         NaN
...                ...         ...         ...
2021-11-09         2.0        10.0         NaN
2021-11-10         3.0         9.0         1.0
2021-11-11         1.0         9.0         NaN
2021-11-12         1.0        10.0         NaN
2021-11-13         1.0         6.0         NaN

[331 rows x 3 columns]

## 뉴발란스 992
그레이   
네이비   
블랙그레이   
화이트실버    

In [121]:
shop992 = pd.read_csv('./data/NAVER-shopping/shopping_new992.csv')
shop992

날짜  뉴발란스992그레이  뉴발란스992네이비  뉴발란스992블랙그레이  뉴발란스992화이트실버
0    2020-11-13          40         6.0           1.0           2.0
1    2020-11-14          40         9.0           NaN           NaN
2    2020-11-15          66         1.0           NaN           NaN
3    2020-11-16          47         2.0           2.0           NaN
4    2020-11-17          65        12.0           NaN           NaN
..          ...         ...         ...           ...           ...
361  2021-11-09          45         5.0           7.0          10.0
362  2021-11-10          45        10.0          14.0           5.0
363  2021-11-11          57         6.0          13.0           1.0
364  2021-11-12          35        10.0          11.0           2.0
365  2021-11-13          44         8.0          10.0           1.0

[366 rows x 5 columns]

In [123]:
shop992.index = shop992['날짜']
shop992 = shop992.drop(['날짜'], axis=1)
shop992

뉴발란스992그레이  뉴발란스992네이비  뉴발란스992블랙그레이  뉴발란스992화이트실버
날짜                                                            
2020-11-13          40         6.0           1.0           2.0
2020-11-14          40         9.0           NaN           NaN
2020-11-15          66         1.0           NaN           NaN
2020-11-16          47         2.0           2.0           NaN
2020-11-17          65        12.0           NaN           NaN
...                ...         ...           ...           ...
2021-11-09          45         5.0           7.0          10.0
2021-11-10          45        10.0          14.0           5.0
2021-11-11          57         6.0          13.0           1.0
2021-11-12          35        10.0          11.0           2.0
2021-11-13          44         8.0          10.0           1.0

[366 rows x 4 columns]

In [124]:
shop992.to_csv('./data/NAVER-shopping/shop_new992.csv')

## 뉴발란스 993
그레이    
네이비    
블랙    

In [125]:
shop993 = pd.read_csv('./data/NAVER-shopping/shopping_new993.csv')
shop993

날짜  뉴발란스993네이비  뉴발란스993그레이  뉴발란스993블랙
0    2020-11-13        12.0          20       16.0
1    2020-11-14         7.0          15        9.0
2    2020-11-15         9.0          25       20.0
3    2020-11-16        10.0          28       22.0
4    2020-11-17         6.0          24       11.0
..          ...         ...         ...        ...
361  2021-11-09         3.0          44       14.0
362  2021-11-10         3.0          36       13.0
363  2021-11-11         6.0          40       12.0
364  2021-11-12         6.0          33       11.0
365  2021-11-13         8.0          33        9.0

[366 rows x 4 columns]

In [126]:
shop993.index = shop993['날짜']
shop993 = shop993.drop(['날짜'], axis=1)
shop993

뉴발란스993네이비  뉴발란스993그레이  뉴발란스993블랙
날짜                                           
2020-11-13        12.0          20       16.0
2020-11-14         7.0          15        9.0
2020-11-15         9.0          25       20.0
2020-11-16        10.0          28       22.0
2020-11-17         6.0          24       11.0
...                ...         ...        ...
2021-11-09         3.0          44       14.0
2021-11-10         3.0          36       13.0
2021-11-11         6.0          40       12.0
2021-11-12         6.0          33       11.0
2021-11-13         8.0          33        9.0

[366 rows x 3 columns]

In [127]:
shop993.to_csv('./data/NAVER-shopping/shop_new993.csv')

## 조던1 로우
스타피쉬    
울프그레이    
트레비스 스캇    

In [128]:
shop_jordanlow = pd.read_csv('./data/NAVER-shopping/shopping_jordan1_low.csv')
shop_jordanlow

날짜  조던1로우스타피쉬  조던1로우울프그레이  조던1로우트레비스스캇
0    2020-12-01        NaN         NaN          2.0
1    2021-01-03        NaN         NaN          3.0
2    2021-01-12        NaN         NaN          2.0
3    2021-01-19        NaN         NaN         11.0
4    2021-02-20        NaN         NaN          3.0
..          ...        ...         ...          ...
157  2021-11-09       11.0        39.0          NaN
158  2021-11-10        NaN        21.0          NaN
159  2021-11-11        NaN        21.0          NaN
160  2021-11-12        2.0         6.0          NaN
161  2021-11-13        5.0        15.0          NaN

[162 rows x 4 columns]

In [129]:
shop_jordanlow.index = shop_jordanlow['날짜']
shop_jordanlow = shop_jordanlow.drop(['날짜'], axis=1)
shop_jordanlow

조던1로우스타피쉬  조던1로우울프그레이  조던1로우트레비스스캇
날짜                                            
2020-12-01        NaN         NaN          2.0
2021-01-03        NaN         NaN          3.0
2021-01-12        NaN         NaN          2.0
2021-01-19        NaN         NaN         11.0
2021-02-20        NaN         NaN          3.0
...               ...         ...          ...
2021-11-09       11.0        39.0          NaN
2021-11-10        NaN        21.0          NaN
2021-11-11        NaN        21.0          NaN
2021-11-12        2.0         6.0          NaN
2021-11-13        5.0        15.0          NaN

[162 rows x 3 columns]

In [130]:
shop_jordanlow.to_csv('./data/NAVER-shopping/shop_jordan1_low.csv')

## 조던1 미드
그레이포크    
스모크그레이     
울프그레이     
짐레드     

In [131]:
shop_jordanmid = pd.read_csv('./data/NAVER-shopping/shopping_jordan1_mid.csv')
shop_jordanmid

날짜  조던1미드그레이포크  조던1미드스모크그레이  조던1미드울프그레이  조던1미드짐레드
0    2020-11-13         NaN         66.0         NaN       NaN
1    2020-11-14         NaN         11.0         NaN       NaN
2    2020-11-15         NaN         15.0         NaN       NaN
3    2020-11-16         NaN         33.0         NaN       NaN
4    2020-11-17         NaN          9.0         NaN       NaN
..          ...         ...          ...         ...       ...
360  2021-11-09         NaN          5.0         6.0      14.0
361  2021-11-10         NaN         10.0        20.0      27.0
362  2021-11-11         NaN          9.0         5.0      30.0
363  2021-11-12         NaN          6.0         3.0      20.0
364  2021-11-13         NaN         11.0        19.0      15.0

[365 rows x 5 columns]

In [132]:
shop_jordanmid.index = shop_jordanmid['날짜']
shop_jordanmid = shop_jordanmid.drop(['날짜'], axis=1)
shop_jordanmid

조던1미드그레이포크  조던1미드스모크그레이  조던1미드울프그레이  조던1미드짐레드
날짜                                                       
2020-11-13         NaN         66.0         NaN       NaN
2020-11-14         NaN         11.0         NaN       NaN
2020-11-15         NaN         15.0         NaN       NaN
2020-11-16         NaN         33.0         NaN       NaN
2020-11-17         NaN          9.0         NaN       NaN
...                ...          ...         ...       ...
2021-11-09         NaN          5.0         6.0      14.0
2021-11-10         NaN         10.0        20.0      27.0
2021-11-11         NaN          9.0         5.0      30.0
2021-11-12         NaN          6.0         3.0      20.0
2021-11-13         NaN         11.0        19.0      15.0

[365 rows x 4 columns]

In [133]:
shop_jordanmid.to_csv('./data/NAVER-shopping/shop_jordan1_mid.csv')

## 조던1 하이
다크모카     
스모크그레이     
하이퍼로얄

In [134]:
shop_jordanhigh = pd.read_csv('./data/NAVER-shopping/shopping_jordan1_high.csv')
shop_jordanhigh

날짜  조던1하이다크모카  조던1하이스모크그레이  조던1하이하이퍼로얄
0    2020-11-13       61.0          6.0         NaN
1    2020-11-14        6.0          NaN         NaN
2    2020-11-15       13.0          NaN         NaN
3    2020-11-16       32.0          NaN         NaN
4    2020-11-17        NaN          6.0         NaN
..          ...        ...          ...         ...
285  2021-11-07       13.0          NaN         NaN
286  2021-11-08        NaN          6.0        10.0
287  2021-11-09        NaN         35.0         NaN
288  2021-11-12        3.0          NaN         NaN
289  2021-11-13       10.0         23.0         NaN

[290 rows x 4 columns]

In [135]:
shop_jordanhigh.index = shop_jordanhigh['날짜']
shop_jordanhigh = shop_jordanhigh.drop(['날짜'], axis=1)
shop_jordanhigh

조던1하이다크모카  조던1하이스모크그레이  조던1하이하이퍼로얄
날짜                                            
2020-11-13       61.0          6.0         NaN
2020-11-14        6.0          NaN         NaN
2020-11-15       13.0          NaN         NaN
2020-11-16       32.0          NaN         NaN
2020-11-17        NaN          6.0         NaN
...               ...          ...         ...
2021-11-07       13.0          NaN         NaN
2021-11-08        NaN          6.0        10.0
2021-11-09        NaN         35.0         NaN
2021-11-12        3.0          NaN         NaN
2021-11-13       10.0         23.0         NaN

[290 rows x 3 columns]

In [136]:
shop_jordanhigh.to_csv('./data/NAVER-shopping/shop_jordan1_high.csv')

## 네이버 데이터랩 쇼핑인사이트 통합 데이터셋

In [139]:
shop_dunk = pd.merge(shopping_dunklow, sample1, left_index=True, right_index=True, how='outer')
shop_dunk

나이키덩크로우골든로드  나이키덩크로우라이트본  나이키덩크로우바시티그린  나이키덩크로우범고래  \
날짜                                                               
2020-11-13          NaN          NaN           NaN         NaN   
2020-11-15          NaN          NaN           NaN         NaN   
2020-11-17          NaN          NaN           NaN         NaN   
2020-11-18          NaN          NaN           NaN         NaN   
2020-11-20          NaN          NaN           NaN         NaN   
...                 ...          ...           ...         ...   
2021-11-09          NaN          4.0           NaN         8.0   
2021-11-10          NaN          3.0           NaN         6.0   
2021-11-11          NaN          3.0           NaN         7.0   
2021-11-12          NaN          2.0           NaN         6.0   
2021-11-13          NaN          2.0           NaN         6.0   

            나이키덩크로우유니버시티블루  나이키덩크로우코스트  나이키덩크하이네이비  나이키덩크하이범고래  나이키덩크하이오렌지  
날짜                                                                          
2020-11-13             NaN         NaN         NaN         NaN         NaN  
2020-11-15             NaN         NaN         NaN         NaN         NaN  
2020-11-17             NaN         NaN         NaN         NaN         NaN  
2020-11-18             NaN         NaN         NaN         NaN         NaN  
2020-11-20             NaN         NaN         1.0         NaN         NaN  
...                    ...         ...         ...         ...         ...  
2021-11-09             NaN        10.0         2.0        10.0         NaN  
2021-11-10             NaN         9.0         3.0         9.0         1.0  
2021-11-11             NaN        20.0         1.0         9.0         NaN  
2021-11-12             NaN        11.0         1.0        10.0         NaN  
2021-11-13             NaN         3.0         1.0         6.0         NaN  

[352 rows x 9 columns]

In [144]:
shop_jordan1 = pd.merge(shop_jordanhigh, shop_jordanmid, left_index=True, right_index=True, how='outer')
shop_jordan1 = pd.merge(shop_jordan1, shop_jordanlow, left_index=True, right_index=True, how='outer')
shop_jordan1

조던1하이다크모카  조던1하이스모크그레이  조던1하이하이퍼로얄  조던1미드그레이포크  조던1미드스모크그레이  \
날짜                                                                        
2020-11-13       61.0          6.0         NaN         NaN         66.0   
2020-11-14        6.0          NaN         NaN         NaN         11.0   
2020-11-15       13.0          NaN         NaN         NaN         15.0   
2020-11-16       32.0          NaN         NaN         NaN         33.0   
2020-11-17        NaN          6.0         NaN         NaN          9.0   
...               ...          ...         ...         ...          ...   
2021-11-09        NaN         35.0         NaN         NaN          5.0   
2021-11-10        NaN          NaN         NaN         NaN         10.0   
2021-11-11        NaN          NaN         NaN         NaN          9.0   
2021-11-12        3.0          NaN         NaN         NaN          6.0   
2021-11-13       10.0         23.0         NaN         NaN         11.0   

            조던1미드울프그레이  조던1미드짐레드  조던1로우스타피쉬  조던1로우울프그레이  조던1로우트레비스스캇  
날짜                                                                    
2020-11-13         NaN       NaN        NaN         NaN          NaN  
2020-11-14         NaN       NaN        NaN         NaN          NaN  
2020-11-15         NaN       NaN        NaN         NaN          NaN  
2020-11-16         NaN       NaN        NaN         NaN          NaN  
2020-11-17         NaN       NaN        NaN         NaN          NaN  
...                ...       ...        ...         ...          ...  
2021-11-09         6.0      14.0       11.0        39.0          NaN  
2021-11-10        20.0      27.0        NaN        21.0          NaN  
2021-11-11         5.0      30.0        NaN        21.0          NaN  
2021-11-12         3.0      20.0        2.0         6.0          NaN  
2021-11-13        19.0      15.0        5.0        15.0          NaN  

[366 rows x 10 columns]

In [145]:
shop_newbalance = pd.merge(shop992, shop993, left_index=True, right_index=True, how='outer')
shop_newbalance

뉴발란스992그레이  뉴발란스992네이비  뉴발란스992블랙그레이  뉴발란스992화이트실버  뉴발란스993네이비  \
날짜                                                                           
2020-11-13          40         6.0           1.0           2.0        12.0   
2020-11-14          40         9.0           NaN           NaN         7.0   
2020-11-15          66         1.0           NaN           NaN         9.0   
2020-11-16          47         2.0           2.0           NaN        10.0   
2020-11-17          65        12.0           NaN           NaN         6.0   
...                ...         ...           ...           ...         ...   
2021-11-09          45         5.0           7.0          10.0         3.0   
2021-11-10          45        10.0          14.0           5.0         3.0   
2021-11-11          57         6.0          13.0           1.0         6.0   
2021-11-12          35        10.0          11.0           2.0         6.0   
2021-11-13          44         8.0          10.0           1.0         8.0   

            뉴발란스993그레이  뉴발란스993블랙  
날짜                                 
2020-11-13          20       16.0  
2020-11-14          15        9.0  
2020-11-15          25       20.0  
2020-11-16          28       22.0  
2020-11-17          24       11.0  
...                ...        ...  
2021-11-09          44       14.0  
2021-11-10          36       13.0  
2021-11-11          40       12.0  
2021-11-12          33       11.0  
2021-11-13          33        9.0  

[366 rows x 7 columns]

In [147]:
NAVER_shop = pd.merge(shop_dunk, shop_jordan1, left_index=True, right_index=True, how='outer')
NAVER_shop = pd.merge(NAVER_shop, shop_newbalance, left_index=True, right_index=True, how='outer')
NAVER_shop

나이키덩크로우골든로드  나이키덩크로우라이트본  나이키덩크로우바시티그린  나이키덩크로우범고래  \
날짜                                                               
2020-11-13          NaN          NaN           NaN         NaN   
2020-11-14          NaN          NaN           NaN         NaN   
2020-11-15          NaN          NaN           NaN         NaN   
2020-11-16          NaN          NaN           NaN         NaN   
2020-11-17          NaN          NaN           NaN         NaN   
...                 ...          ...           ...         ...   
2021-11-09          NaN          4.0           NaN         8.0   
2021-11-10          NaN          3.0           NaN         6.0   
2021-11-11          NaN          3.0           NaN         7.0   
2021-11-12          NaN          2.0           NaN         6.0   
2021-11-13          NaN          2.0           NaN         6.0   

            나이키덩크로우유니버시티블루  나이키덩크로우코스트  나이키덩크하이네이비  나이키덩크하이범고래  나이키덩크하이오렌지  \
날짜                                                                           
2020-11-13             NaN         NaN         NaN         NaN         NaN   
2020-11-14             NaN         NaN         NaN         NaN         NaN   
2020-11-15             NaN         NaN         NaN         NaN         NaN   
2020-11-16             NaN         NaN         NaN         NaN         NaN   
2020-11-17             NaN         NaN         NaN         NaN         NaN   
...                    ...         ...         ...         ...         ...   
2021-11-09             NaN        10.0         2.0        10.0         NaN   
2021-11-10             NaN         9.0         3.0         9.0         1.0   
2021-11-11             NaN        20.0         1.0         9.0         NaN   
2021-11-12             NaN        11.0         1.0        10.0         NaN   
2021-11-13             NaN         3.0         1.0         6.0         NaN   

            조던1하이다크모카  조던1하이스모크그레이  조던1하이하이퍼로얄  조던1미드그레이포크  조던1미드스모크그레이  \
날짜                                                                        
2020-11-13       61.0          6.0         NaN         NaN         66.0   
2020-11-14        6.0          NaN         NaN         NaN         11.0   
2020-11-15       13.0          NaN         NaN         NaN         15.0   
2020-11-16       32.0          NaN         NaN         NaN         33.0   
2020-11-17        NaN          6.0         NaN         NaN          9.0   
...               ...          ...         ...         ...          ...   
2021-11-09        NaN         35.0         NaN         NaN          5.0   
2021-11-10        NaN          NaN         NaN         NaN         10.0   
2021-11-11        NaN          NaN         NaN         NaN          9.0   
2021-11-12        3.0          NaN         NaN         NaN          6.0   
2021-11-13       10.0         23.0         NaN         NaN         11.0   

            조던1미드울프그레이  조던1미드짐레드  조던1로우스타피쉬  조던1로우울프그레이  조던1로우트레비스스캇  \
날짜                                                                     
2020-11-13         NaN       NaN        NaN         NaN          NaN   
2020-11-14         NaN       NaN        NaN         NaN          NaN   
2020-11-15         NaN       NaN        NaN         NaN          NaN   
2020-11-16         NaN       NaN        NaN         NaN          NaN   
2020-11-17         NaN       NaN        NaN         NaN          NaN   
...                ...       ...        ...         ...          ...   
2021-11-09         6.0      14.0       11.0        39.0          NaN   
2021-11-10        20.0      27.0        NaN        21.0          NaN   
2021-11-11         5.0      30.0        NaN        21.0          NaN   
2021-11-12         3.0      20.0        2.0         6.0          NaN   
2021-11-13        19.0      15.0        5.0        15.0          NaN   

            뉴발란스992그레이  뉴발란스992네이비  뉴발란스992블랙그레이  뉴발란스992화이트실버  뉴발란스993네이비  \
날짜                                                                           
2020-11-13          40         6.0           1.0           2.0       

In [154]:
NAVER_shop.to_csv('./data/NAVER-shopping/NAVER_shop.csv')

In [153]:
NAVER_trend

조던1 하이 다크모카  조던1 하이 스모크그레이  조던1 하이 하이퍼로얄  조던1 미드 그레이포크  \
날짜                                                                   
2020-11-01      0.00000        7.14285           0.0             0   
2020-11-02      0.00000        9.52380           0.0             0   
2020-11-03      0.00000        5.95238           0.0             0   
2020-11-04      8.33333        7.14285           0.0             0   
2020-11-05      0.00000        0.00000           0.0             0   
...                 ...            ...           ...           ...   
2021-11-09      0.00000        0.00000           0.0             0   
2021-11-10      0.00000        0.00000           0.0             0   
2021-11-11      0.00000        0.00000           0.0             0   
2021-11-12      0.00000        0.00000           0.0             0   
2021-11-13      0.00000        0.00000           0.0             0   

            조던1 미드 스모크그레이  조던1 미드 울프그레이  조던1 미드 짐레드  조던1 로우 스타피쉬  \
날짜                                                                 
2020-11-01        3.46938       0.00000     0.00000      0.00000   
2020-11-02        3.57142       0.00000     0.61224      0.00000   
2020-11-03        2.85714       0.00000     0.91836      0.00000   
2020-11-04        3.67346       0.00000     0.51020      0.00000   
2020-11-05        2.95918       0.00000     0.51020      0.00000   
...                   ...           ...         ...          ...   
2021-11-09        2.24489       0.81632     4.28571      3.60205   
2021-11-10        1.63265       3.16326     4.18367      2.74442   
2021-11-11        2.14285       1.12244     4.69387      3.08747   
2021-11-12        2.24489       1.02040     3.67346      4.11663   
2021-11-13        2.44897       1.93877     6.32653      4.63121   

            조던1 로우 울프그레이  조던1 로우 트레비스 스캇  나이키 덩크하이 네이비  나이키 덩크하이 범고래  \
날짜                                                                     
2020-11-01       0.00000             0.0       0.00000       0.00000   
2020-11-02       0.00000             0.0       0.00000       0.00000   
2020-11-03       0.00000             0.0       0.00000       0.00000   
2020-11-04       0.00000             0.0       0.00000       0.00000   
2020-11-05       0.00000             0.0       0.00000       0.00000   
...                  ...             ...           ...           ...   
2021-11-09       1.20068             0.0       0.45248       5.07792   
2021-11-10       1.02915             0.0       0.85470       4.12267   
2021-11-11       1.02915             0.0       0.65359       4.47461   
2021-11-12       2.74442             0.0       0.60331       5.12820   
2021-11-13       1.71526             0.0       0.55304       3.92156   

            나이키 덩크하이 오렌지  나이키 덩크 로우 골든로드   나이키 덩크 로우 라이트본  나이키 덩크 로우 바시티그린  \
날짜                                                                           
2020-11-01           0.0          0.00000         0.00000          0.00000   
2020-11-02           0.0          0.00000         0.00000          0.00000   
2020-11-03           0.0          0.00000         0.00000          0.00000   
2020-11-04           0.0          0.00000         0.00000          0.00000   
2020-11-05           0.0          0.00000         0.00000          0.00000   
...                  ...              ...             ...              ...   
2021-11-09           0.0          0.18141         4.95011          0.12526   
2021-11-10           0.0          0.18141         4.80324          0.07775   
2021-11-11           0.0          0.14686         4.68230          0.11662   
2021-11-12           0.0          0.22029         3.79681          0.06479   
2021-11-13           0.0          0.13390         3.54196          0.09934   

            나이키 덩크 로우 범고래  나이키 덩크 로우 유니버시티블루  나이키 덩크로우 코스트  뉴발란스 992 그레이  \
날짜                                                                         
2020-11-01        0.00000            0.00000       0.00000       7.47217   
2020-11-02        0.00000           